python/cuml/fil/fil.pyx

```diff
@@ -574,6 +575,15 @@ class ForestInference(Base):
             model passed.
 
         """
+        if(has_treelite() is False):
+            raise ImportError("Treelite needs to be built from source"
+                              " for this function to be used.")
+        else:
+            if(treelite.__version__ <= '0.32'):
```

Comment:
Should do a version comparison if possible. I think we already have a package requires that we use elsewhere?